requirement.txt

```diff
@@ -1,6 +1,6 @@
 ostorlab[agent]
 rich
-requests
```

Comment:
Why?

Reply:
In Ostorlab, we pin the version of requests to `2.26.0`. and when we try to install requests again from the requirements, it will pick the version that has already been installed, which is `2.26.0`. However, this version does not support `JSONDecodeError`, which is why if you see the version of requests inside the `asteroid` container you will find it is `2.26.0` not the `latest` version